.github/actions/compile-index/lib/BsdataGallery/BsdataGallery.psm1

```diff
@@ -19,9 +19,10 @@ function Get-BsdataGalleryCatpkg {
     battleScribeVersion = ($entries.cache.catpkg.properties.battleScribeVersion | Sort-Object -Bottom 1) -as [string]
   } + $GallerySettings.urls + @{
     repositories = @($entries.cache | ForEach-Object {
-        $_.catpkg.properties.archived = $_.repo.archived -eq $true
-        return $_.catpkg.properties
-      })
+      $_.catpkg.properties.archived = $_.repo.archived -eq $true
+      $_.catpkg.properties.lastUpdated = [datetime]::ParseExact($_.catpkg.properties.lastUpdated, 'MM/dd/yyyy HH:mm:ss', $null).ToString("yyyy'-'MM'-'dd'T'HH':'mm':'ss'.'fff+0000")
```

Comment:
The `$_.catpkg.properties.lastUpdated` should already be a `DateTime`, so all that's left to do is just the `ToString` part.

📝 I tested it locally with arbitrary values and it'd still need to be tested with that specific API responses and script.